roles/confluent.kafka_connect/tasks/confluent_hub.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Creating Confluent Hub Install Dir
+  file:
+    path: "{{kafka_connect_confluent_hub_client.install_dir}}"
+    state: directory
+    group: "{{kafka_connect_group}}"
+    owner: "{{kafka_connect_user}}"
+    mode: '764'
+  
+- name: Installing/Unarchiving Confluent Hub
+  unarchive:
```

Comment:
to make this idempopent- probably want a creates
https://docs.ansible.com/ansible/latest/modules/unarchive_module.html